.github/container/Dockerfile.jax

```diff
@@ -16,19 +18,23 @@ ARG BUILD_DATE
 ###############################################################################
 
 FROM ${BASE_IMAGE} as builder
-ARG DEST_MANIFEST_DIR
+ARG GIT_URLREF_JAX
+ARG GIT_URLREF_XLA
 ARG SRC_PATH_JAX
 ARG SRC_PATH_XLA
 ARG BAZEL_CACHE
-ARG GIT_USER_NAME
-ARG GIT_USER_EMAIL
 
-ADD --chmod=777 create-distribution.sh ${DEST_MANIFEST_DIR}/
+RUN <<"EOF" bash -exu -o pipefail
+DEFAULT_GIT_URLREF_JAX=$(yq '.jax | [(.url, .latest_verified_commit)] | join("#")' ${MANIFEST_FILE})
```

Comment:
I agree that the `yq` command gets repeated many times appears redundant. However, it is a small and simple snippet that is easy to understand, and not abstracting it away means that we make it more explicit about what happens. Taken one step back, we can introduce a simple bash function to handle it rather than burying it inside `get-source.sh`. The goal is to make sure the system is as loosely decoupled as possible.